gclient/src/api/storage/block.rs

```diff
@@ -124,12 +121,12 @@ impl GearApi {
     }
 
     pub async fn queue_processing_stopped(&self) -> Result<bool> {
-        let at = storage().gear().queue_state();
+        let at = storage().gear().execute_inherent();
         self.0
             .storage()
             .fetch(&at, None)
             .await?
             .ok_or(Error::StorageNotFound)
```

Comment:
Seems your ran into this error, cause default value set for a storage - so such key does not exist in database. here should be `unwrap_or(true)` instead of matching Option as returned value